packages/react-provider/src/components/FluentProvider/renderFluentProvider.tsx

```diff
@@ -16,9 +17,11 @@ export const renderFluentProvider = (state: FluentProviderState) => {
   return (
     <ThemeProvider {...state} theme={theme}>
       <ProviderContext.Provider value={value}>
-        <TabsterProvider document={document} dir={dir}>
-          {state.children}
-        </TabsterProvider>
+        <TooltipProvider>
```

Comment:
shouldn't this be rendered conditionally or lazily ? `FluentProvider` needs to be rendered for v9, which means that you will always get tooltip even if you don't want it in your bundle

Reply:
Resolving in favour of @layershifter's comment